packages/terra-toolkit-docs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated terra-functional-testing documentation to add additional information to the about page and individual services
+
 * Added
   * Added documentation in terra-functional-testing to describe how visual regression determines screenshot dimensions.
 
```

Comment:
Make sure to expand the About page below this file to see the changes there. It's collapsed by default